src/Components/Artwork/GridItem.tsx

```diff
@@ -96,12 +96,27 @@ export const ArtworkGridItem: React.FC<ArtworkGridItemProps> = ({
               preventRightClick={!isTeam}
             />
           ) : (
-            <ResponsiveBox
-              aspectWidth={4}
-              aspectHeight={3}
-              position="relative"
-              maxWidth="100%"
-            />
+            <>
+              <ResponsiveBox
+                aspectWidth={4}
+                aspectHeight={3}
+                position="relative"
+                maxWidth="100%"
+              />
+              {!!isMyCollectionArtwork && (
```

Comment:
I wonder if this could be the default behavior of this component.  Removing the `isMyCollectionArtwork`check shouldn't affect other artwork grids because all artworks outside of My Collection should have images.